docset/winserver2025-ps/smbshare/Get-SmbServerCertProps.md

```diff
@@ -11,27 +11,31 @@ title: Get-SmbServerCertProps
 # Get-SmbServerCertProps
 
 ## SYNOPSIS
-Retrieves an SMB over QUIC-mapped certificate’s properties and tests certificate validity.
+Retrieves an SMB over QUIC-mapped certificate's properties and tests certificate validity.
 
 ## SYNTAX
 
 ```
-Get-SmbServerCertProps `-Name` <String> `-Force` <CommonParameters>
+Get-SmbServerCertProps -Name <String> -Force <CommonParameters>
 ```
 
 ## DESCRIPTION
-The **Get-SmbServerCertProps** cmdlet retrieves the properties of a certificate associated with the SMB server for SMB over QUIC on Windows Server 2022 Datacenter: Azure Edition. It also tests the validity of the certificate. This cmdlet is not used for Windows or other Windows Server editions. For more information, review [SMB over QUIC](https://aka.ms/smboverquic).
+
+The `Get-SmbServerCertProps` cmdlet retrieves the properties of a certificate associated with the
+SMB server for SMB over QUIC. For more information, see [SMB over QUIC](https://aka.ms/smboverquic).
 
 ## EXAMPLES
 
 ### Example 1 - Retrieve the properties of a certificate associated with the SMB server
 
-This command retrieves the properties and validity of the `2022-ae-02.corp.contoso.com` SMB over QUIC endpoint certificate mapping.
+This command retrieves the properties and validity of the `2022-ae-02.corp.contoso.com` SMB over
```

Comment:
set everywhere to "2025-srv-01.corp.contoso.com", it implies more strongly that QUIC now supports non-azure edition inWS2025